Dockerfile.QA

```diff
@@ -371,3 +371,6 @@ ENV LD_LIBRARY_PATH /opt/tritonserver/qa/clients:${LD_LIBRARY_PATH}
 
 # DLIS-3631: Needed to run Perf Analyzer CI tests correctly
 ENV LD_LIBRARY_PATH /opt/hpcx/ompi/lib:${LD_LIBRARY_PATH}
+
+# Required for PyTorch to pickup the correct HPCX libraries
```

Comment:
should we just combine this with the previous line and add a comment?

Reply:
I think it would be a bit more difficult to comment on which sections are added for PA/PyTorch.